src/Microsoft.Health.Fhir.Shared.Client/FhirClient.cs

```diff
@@ -283,6 +283,19 @@ public async Task<Uri> ExportAsync(string path = "", string parameters = "", Can
             return response.Content.Headers.ContentLocation;
         }
 
+        public async Task<Uri> AnonymizedExportAsync(string anonymizationConfig, string etag = null, CancellationToken cancellationToken = default)
+        {
+            using var message = new HttpRequestMessage(HttpMethod.Get, $"$export?_anonymization_config={anonymizationConfig}&_anonymization_config_etag={etag}");
```

Comment:
Do the query string parameter arguments need to be URL encoded first?

Reply:
it support both original string and encoded url.